script.js

```diff
@@ -0,0 +1,159 @@
+let my_resume = {
```

Comment:
This file is a bit hard for me to follow, seems overly verbose. Looks like the you're doing this to target an area and place the data? Splitting this data into multiple components, would likely be a lot easier for you, especially to handle dynamic data.